flash/text/question_answering/finetuning.py

```diff
@@ -1,93 +0,0 @@
-# Copyright The PyTorch Lightning team.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from torch.nn import Module
-
-
-def _get_question_answering_bacbones_for_freezing(pl_module: Module):
-    model_type = pl_module.config.model_type
-
-    if model_type in ["albert", "reformer"]:
-        if model_type == "albert":
-            model = pl_module.albert
-        elif model_type == "reformer":
-            model = pl_module.reformer
-
-    elif model_type in ["bart", "bigbird_pegasus", "led", "mbart"]:
-        if model_type in ["bart", "bigbird_pegasus", "mbart"]:
-            model = pl_module.model
-        elif model_type == "led":
-            model = pl_module.led
-
-    elif model_type == [
```

Comment:
@krshrimali The equals sign here was a typo as `model_type` would be a string (should have been `in` for the question answering freezing to work correctly)

Reply:
Got it! Good catch